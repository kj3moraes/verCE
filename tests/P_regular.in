def add(x y) x + y
h-j = 2
extern sin(arg)
