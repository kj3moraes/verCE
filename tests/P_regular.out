DEF:def 
LPAREN:( 
ID:x 
RPAREN:) 
ID:x 
PLUS:+ 
ID:y 
ID:h 
MINUS:- 
ID:j 
BECOMES:= 
NUM:2 
EXTERN:extern 
NUM:12 
ID:f5 
